interfaces/builtin/home.go

```diff
@@ -85,13 +86,13 @@ audit deny @{HOME}/bin wl,
 const homeConnectedPlugAppArmorWithAllRead = `
 # Allow non-owner read to non-hidden and non-snap files and directories
 capability dac_read_search,
-@{HOME}/               r,
-@{HOME}/[^s.]**        r,
-@{HOME}/s[^n]**        r,
-@{HOME}/sn[^a]**       r,
-@{HOME}/sna[^p]**      r,
-@{HOME}/snap[^/]**     r,
-@{HOME}/{s,sn,sna}{,/} r,
+###PROMPT###@{HOME}/               r,
+###PROMPT###@{HOME}/[^s.]**        r,
+###PROMPT###@{HOME}/s[^n]**        r,
+###PROMPT###@{HOME}/sn[^a]**       r,
+###PROMPT###@{HOME}/sna[^p]**      r,
+###PROMPT###@{HOME}/snap[^/]**     r,
+###PROMPT###@{HOME}/{s,sn,sna}{,/} r,
```

Comment:
Nitpick, perhaps we can just inject the prompt prefix to each non-empty, non-comment line?

Reply:
I believe there there will be some rules (perhaps in other interfaces rather than "home") for which we will not want to prompt. We could default to adding prompt with a `###NOPROMPT###` prefix to skip a given rule, but I prefer the current approach over that, I think.

Reply:
Ah, for that matter, these two rules are omitted from prompting:
```
# Disallow writes to the well-known directory included in
# the user's PATH on several distributions
audit deny @{HOME}/bin/{,**} wl,
audit deny @{HOME}/bin wl,
```
So we can't insert prompting for everything. We have to use something like `###PROMPT###` or `###NOPROMPT###`. I prefer the former.